.reuse/dep5

```diff
@@ -0,0 +1,73 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: OSS Review Toolkit
+Source: https://github.com/heremaps/oss-review-toolkit
+
+# If multiple globs match the same files, the last match wins.
+
+Files: .*
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
+
+Files: */.gitignore
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
+
+Files: *.md
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
+
+Files: *.gradle.kts
+Copyright: 2017-2020 HERE Europe B.V. <opensource@here.com>
+License: Apache-2.0
```

Comment:
I think you can remove this entry after #2102 is merged.

Reply:
Yes, that's was the intention behind that PR.